### PR TITLE
Remove inches from personal website

### PR DESCRIPTION
**Pull request by MaxWofford**
Remove references to inches
Google changed from verb -> noun

### CODE REVIEW

File: workshops/personal_website/README.md [hunk 1]
```diff
@@ -407,10 +407,8 @@ img {
 - **Select** every **`img` tag** on this HTML page and set the **width** to be
   **`200px`**
 
-> **`px`** stands for **pixels**. **Pixels** are the tiny dots that make up a
-> computer screen. By setting the width to **200px**, we are saying we want
-> every image on the page to span the length of 200 pixels (which is about 2
-> inches on a screen). If you want to learn more about pixels, you can Google
+> `px` stands for **pixels**, a unit of size on the screen. If you want to learn
+> more about pixels, you can Google search
 > ["What is a pixel?"](https://www.google.com/#q=what+is+a+pixel)
```
Comment (zachlatta) [inline workshops/personal_website/README.md]: I think having some sort of example for the size of pixels is helpful. Keep in mind these are CSS pixels, which are resolution independent (to my understanding, see http://stackoverflow.com/a/609561/1001686).

Comment (MaxWofford) [inline workshops/personal_website/README.md]: @zachlatta While that info is good for someone doing a deeper dive, I think it's out of the scope of this section of the workshop.

Comment (MaxWofford) [inline workshops/personal_website/README.md]: @zachlatta As for removing the example size (in inches), https://gist.github.com/remixz/3bf575b32bb96be43ea0

Comment (zachlatta) [inline workshops/personal_website/README.md]: I think we could probably pretty easily use some other measurement of size
(width of thumb or just centimeters).

On Wed, Feb 24, 2016 at 10:04 AM, Max Wofford <notifications@github.com>
wrote:

> In workshops/personal_website/README.md
> <https://github.com/hackclub/hackclub/pull/735#discussion_r53979408>:
>
> > @@ -407,10 +407,8 @@ img {
> >  - **Select** every **`img` tag** on this HTML page and set the **width** to be
> >    **`200px`**
> >
> > -> **`px`** stands for **pixels**. **Pixels** are the tiny dots that make up a
> > -> computer screen. By setting the width to **200px**, we are saying we want
> > -> every image on the page to span the length of 200 pixels (which is about 2
> > -> inches on a screen). If you want to learn more about pixels, you can Google
> > +> `px` stands for **pixels**, a unit of size on the screen. If you want to learn
> > +> more about pixels, you can Google search
> >  > ["What is a pixel?"](https://www.google.com/#q=what+is+a+pixel)
>
> @zachlatta <https://github.com/zachlatta> As for removing the example
> size (in inches), https://gist.github.com/remixz/3bf575b32bb96be43ea0
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/735/files#r53979408>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>

Comment (zachlatta) [inline workshops/personal_website/README.md]: If you think so, sounds good to me!

On Wed, Feb 24, 2016 at 10:04 AM, Max Wofford <notifications@github.com>
wrote:

> In workshops/personal_website/README.md
> <https://github.com/hackclub/hackclub/pull/735#discussion_r53979331>:
>
> > @@ -407,10 +407,8 @@ img {
> >  - **Select** every **`img` tag** on this HTML page and set the **width** to be
> >    **`200px`**
> >
> > -> **`px`** stands for **pixels**. **Pixels** are the tiny dots that make up a
> > -> computer screen. By setting the width to **200px**, we are saying we want
> > -> every image on the page to span the length of 200 pixels (which is about 2
> > -> inches on a screen). If you want to learn more about pixels, you can Google
> > +> `px` stands for **pixels**, a unit of size on the screen. If you want to learn
> > +> more about pixels, you can Google search
> >  > ["What is a pixel?"](https://www.google.com/#q=what+is+a+pixel)
>
> @zachlatta <https://github.com/zachlatta> While that info is good for
> someone doing a deeper dive, I think it's out of the scope of this section
> of the workshop.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/735/files#r53979331>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>

Comment (MaxWofford) [inline workshops/personal_website/README.md]: cc @remixz

**Comment by remixz** [comment]
I think just linking to a search about what pixels are is good. My biggest thing with the inches was that it might not be accurate on devices like phones, which people do want to develop things for. I just don't want to accidentally spread misinformation. :smile:

**Comment by zachlatta** [comment]
Agreed with Google being enough.

Quick note though: shouldn't the pixel sizes be the same on mobile (assuming the viewport is set up correctly)?

**Pull request merged**